src/outlet/index.ts

```diff
@@ -121,25 +120,60 @@ export class Outlet {
 						const attestationBlob = this.getDataFromQuery('attestation')
 						const attestationSecret = '0x' + this.getDataFromQuery('requestSecret')
 
-						const issuerConfig = this.getIssuerConfigById(issuer)
-
-						const ticketRecord = await this.ticketStorage.getStoredTicketFromDecodedToken(createIssuerHashArray(issuerConfig), token)
-
-						const useToken = await AuthHandler.getUseToken(issuerConfig, attestationBlob, attestationSecret, ticketRecord)
-
-						if (requesterURL) {
-							const params = new URLSearchParams(requesterURL.hash.substring(1))
-							params.set(this.getCallbackUrlKey('action'), 'proof-callback')
-							params.set(this.getCallbackUrlKey('issuer'), issuer)
-							params.set(this.getCallbackUrlKey('attestation'), useToken.proof as string)
-							params.set(this.getCallbackUrlKey('type'), ticketRecord.type)
-							params.set(this.getCallbackUrlKey('token'), tokenString)
-
-							requesterURL.hash = params.toString()
-
-							window.location.href = requesterURL.href
-
-							return
+						let issuerConfig = this.getIssuerConfigById(issuer) ?? this.tokenConfig
+
+						let useToken
+
+						let localStorageAuthRequest = JSON.parse(localStorage.getItem('token-auth-request'))
```

Comment:
@nicktaras could you add 'token-auth-request' to the default config or make as Constant, defined in some config file? because its value duplicated in 2 positions, someone can change one of them by accident

Reply:
Updated to use **LocalStorageMessaging.TOKEN_AUTH_REQUEST**  in these locations